src/components/data_submission/NIHAdministrativeInformation.js

```diff
@@ -0,0 +1,339 @@
+        }),
+        h(FormField, {
+          id: 'consentProcessDidNotAddressFutureUseOrBroadSharing',
+          defaultValue: initialFormData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.consentProcessDidNotAddressFutureUseOrBroadSharing),
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'The consent process did not specifically address future use or broad data sharing for samples collected after January 25, 2015',
+        }),
+        h(FormField, {
+          id: 'consentProcessInadequatelyAddressesRisk',
+          defaultValue: initialFormData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.consentProcessInadequatelyAddressesRisk),
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'The consent processes inadequately addresses risks related to future use or broad data sharing for samples collected after January 25, 2015',
+        }),
+        h(FormField, {
+          id: 'consentProcessPrecludesFutureUseOrBroadSharing',
+          defaultValue: initialFormData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.consentProcessPrecludesFutureUseOrBroadSharing),
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'The consent specifically precludes future use or broad sharing (including a statement that use of data will be limited to the original researchers)',
+        }),
+        h(FormField, {
+          id: 'otherInformedConsentLimitationsOrConcerns',
+          defaultValue: initialFormData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.otherInformedConsentLimitationsOrConcerns),
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'Other informed consent limitations or concerns',
+        }),
+        h(FormField, {
+          id: 'otherReasonForRequest',
+          defaultValue: initialFormData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.otherReasonForRequest),
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'Other'
+        }),
+      ]),
+      h(FormField, {
+        isRendered: true, // todo: otherReasonForRequest,
+        id: 'alternativeDataSharingPlanExplanation',
+        defaultValue: initialFormData?.alternativeDataSharingPlanExplanation,
+        onChange,
+        title: 'Explanation for request',
+        validators: [FormValidators.REQUIRED]
+      }),
+      h(FormField, {
+        type: FormFieldTypes.FILE,
+        title: 'Upload your alternative sharing plan.',
+        id: 'alternativeDataSharingPlanFile',
+        onChange,
```

Comment:
Would like some guidance on how exactly this should function in conjunction with how the file `FormField` should operate. Right now, it adds the `File` object to `formData.alternativeDataSharingPlanFile`